### DISCUSSION TITLE
matchedGeometryEffect across navigation transitions?

### DISCUSSION THREAD

**Discussion by alexfringes** [davdroman/swiftui-navigation-transitions#69] [Q&A]
Hey, thanks again for providing this super powerful library, with really nice abstractions.
I was wondering what your suggestions would be for moving a subview between navigation destinations, across the transition.
I haven't dug enough through the code yet, but maybe this would require parsing for .matchedGeometryEffects (or a version of that modifier specific to this library), isolating that view in the UIViewControllerTransitioning-facing code, and matching what .matchedGeometryEffect would do if this were a SwiftUI native solution?

**Comment by davdroman** [comment]
This is basically #4. I suspect it's possible but certainly not trivial. It's one of the pieces I'll be digging into soon :)
